tasks/update_nodes.go

```diff
@@ -70,7 +77,7 @@ func (t *UpdateNodesTask) updateUsingRocketscan() error {
 }
 
 func (t *UpdateNodesTask) Run() {
-	ticker := time.NewTicker(time.Duration(300) * time.Second)
+	ticker := time.NewTicker(time.Duration(1) * time.Second)
```

Comment:
very tight loop. maybe 30 seconds? i doubt anyone will request a cred within 30s of registering

Reply:
This should only affect the first "period". The ticker will be reset to either 300 seconds (success case) or 30 seconds (fail case).